query_execution/PolicyEnforcer.hpp

```diff
@@ -60,12 +61,14 @@ class PolicyEnforcer {
                  const std::size_t num_numa_nodes,
                  CatalogDatabaseLite *catalog_database,
                  StorageManager *storage_manager,
-                 tmb::MessageBus *bus)
+                 tmb::MessageBus *bus,
+                 WorkerDirectory *worker_directory)
```

Comment:
Changed the order. BTW, this is not a convention that is followed elsewhere in the file.